pkg/operator/resourcesynccontroller/core.go

```diff
@@ -18,9 +18,6 @@ func CombineCABundleConfigMaps(destinationConfigMap ResourceLocation, lister cor
 	certificates := []*x509.Certificate{}
 	for _, input := range inputConfigMaps {
 		inputConfigMap, err := lister.ConfigMaps(input.Namespace).Get(input.Name)
-		if apierrors.IsNotFound(err) {
-			continue
-		}
```

Comment:
The configmap may be present but with empty "ca-bundle.crt" data key. Which will end up in the same situation as non-existing CM.

@deads2k @mfojtik given you created/approved the PR introducing the function (https://github.com/openshift/library-go/pull/191). Was the assumption here either of the CMs can be ignored if not present? The current implementation gives impression of AnyOf rather than AllOf.